module/VuFind/src/VuFind/View/Helper/Root/LinkifyFactory.php

```diff
@@ -70,8 +71,9 @@ public function __invoke(
 
         $highlighter = new VuFindHighlighter($proxyUrl);
         $encoder = new HtmlSpecialcharsEncoder();
+        $validatorExceptEmail = new Validator(true, [], [], false);
```

Comment:
Maybe this would be clearer (and more future-proof) with a named argument:

```suggestion
        $validatorExceptEmail = new Validator(matchEmails: false);
```

Reply:
Good idea! Done.